homeassistant/components/google_assistant/trait.py

```diff
@@ -823,14 +878,16 @@ def query_attributes(self):
 
         if domain == cover.DOMAIN:
             return {"isRunning": state in (cover.STATE_CLOSING, cover.STATE_OPENING)}
+        if domain == valve.DOMAIN:
+            return {"isRunning": True}
```

Comment:
Why do you consider always `isRunning` as `True`? I think there are 2 possible logics for isRunning to be true:

- when the valve is somehow open
- when the valve is openning or closing

Reply:
We cannot trust the state of the valve. The stop service should be available under any condition.
Further, this is a tweak to asure that there is a `stop` button. Returning `True` would show a `Start` button, and that does not make sense, hence we always return `True`. As you could see, I added a comment on that.

Reply:
I added some screenshots in the PR description.

Reply:
Why should a stop button be available to a closed valve? It also doesn't make sense. Is this a Google UX issue?

Reply:
It is not guaranteed the valve supports a state to tell it is opening or closing.